playbooks/roles/beaver/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
as a note, this was a pattern i started because of the desire to move the roles into galaxy proper at a later date. Not that we should keep that patter, just saying that this is my fault.